### PR TITLE
chore: align cds-rc.json and package.json config

### PR DESCRIPTION
**Pull request by johannes-vogel**
I recently stumbled over the spread configuration within sflight when analyzing an issue.
If the configuration is not at a single place, it's very hard to quickly get an understanding of applied configurations.

I'd propose to only use package.json in sflight (alternatively use profiles).

In addition, the `cdsc` config should rather stay internal I think. @agoerler was there a specific reason to apply them as well? If yes, I think we should fix it.

### CODE REVIEW

**Comment by agoerler** [comment]
> In addition, the `cdsc` config should rather stay internal I think. @agoerler was there a specific reason to apply them as well? If yes, I think we should fix it.

I think we have to keep 

```json
    "cdsc": {
        "betterSqliteSessionVariables": true
    }
```

as by some strange and not-yet-understood side effect this effects the creation of session variables on H2 (sic).

Alternatively, we can remove the [dependency](https://github.com/SAP-samples/cap-sflight/blob/main/package.json#L26) to "@cap-js/sqlite". This also helps.

File: package.json [hunk 1]
```diff
@@ -109,6 +109,10 @@
           }
         ]
       }
+    },
+    "sql": {
+      "transitive_localized_views" : false,
+      "native_hana_associations" : false
```
Comment (agoerler) [inline package.json:115]:
```suggestion
      "native_hana_associations" : false,
      "cdsc": {
        "betterSqliteSessionVariables": true
    }
```

**Review by agoerler — APPROVED** [review]

**Comment by agoerler** [comment]
> > In addition, the `cdsc` config should rather stay internal I think. @agoerler was there a specific reason to apply them as well? If yes, I think we should fix it.
> 
> I think we have to keep
> 
> ```json
>     "cdsc": {
>         "betterSqliteSessionVariables": true
>     }
> ```
> 
> as by some strange and not-yet-understood side effect this effects the creation of session variables on H2 (sic).
> 
> Alternatively, we can remove the [dependency](https://github.com/SAP-samples/cap-sflight/blob/main/package.json#L26) to "@cap-js/sqlite". This also helps.

I have double-checked. This issue has vanished. Hence we can do without `"betterSqliteSessionVariables": true`.

File: .cdsrc.json [hunk 2]
```diff
@@ -1,11 +1 @@
-{
-    "sql": {
-        "transitive_localized_views" : false,
-        "native_hana_associations" : false
-    },
-    "cdsc": {
-        "fewerLocalizedViews": true,
-        "betterSqliteSessionVariables": true,
```
Comment (stewsk) [inline .cdsrc.json:8]: @johannes-vogel Isn't that one needed for the node runtime with the new sqlite DB service?

Comment (agoerler) [reply .cdsrc.json:8]:
```sh
cds deploy --dry -2 sqlite

->

... 

CREATE VIEW localized_sap_fe_cap_travel_SupplementType AS SELECT
  coalesce(localized_1.name, L_0.name) AS name,
  coalesce(localized_1.descr, L_0.descr) AS descr,
  L_0.code
FROM (sap_fe_cap_travel_SupplementType AS L_0 LEFT JOIN sap_fe_cap_travel_SupplementType_texts AS localized_1 ON localized_1.code = L_0.code AND localized_1.locale = session_context( '$user.locale' )); 

...
```

Looks good to me.

Also `cds run` starts a CAP Node.js app, which looks OK to me.
```

**Pull request merged**